src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -359,11 +378,13 @@ public void traverse() {
         for (int i = 0, batchCount = 1; i < sampleCount; i += updatedBatchSize, ++batchCount) {
 
             final Map<String, FeatureReader<VariantContext>> sampleToReaderMap =
-                getFeatureReaders(new ArrayList<>(sampleNameToVcfUri.keySet()), sampleNameToVcfUri, updatedBatchSize, sampleCount, i);
+                    headerLoadingExecutorService != null
+                            ? getFeatureReaders( sampleNameToVcfPath, updatedBatchSize, i)
+                            : getFeatureReadersWithoutMultithreading( sampleNameToVcfPath, updatedBatchSize, i);
 
             logger.info("Importing batch " + batchCount + " with " + sampleToReaderMap.size() + " samples");
             final long variantContextBufferSize = vcfBufferSizePerSample * sampleToReaderMap.size();
-
+            logger.debug("Starting batch import: " + System.currentTimeMillis());
```

Comment:
@droazen All of our log messages are timestamped already.  Should I just ditch the times?